### ISSUE TITLE
Low risk message during active infection

### ISSUE THREAD

**Issue by heinp** [corona-warn-app/cwa-app-android#4757]
## Avoid duplicates
* [ x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ x] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [x ] Bug is not already reported in another issue

## Technical details

- Device name: Huawei P30 Pro
- Android version: 10
- EMUI Version: 11
- App version: 2.16.2

## Describe the bug

Push notification of lowered risk during active infection. Might be confusing. 

## Steps to reproduce the issue

1. Get red warning
2. Get infected and activate positive test in App
3. Let 14 days pass after step 1

## Expected behaviour

Now message of "low risk" during infection. 

## Possible Fix

Check infection status before sending message.

**Comment by Ein-Tim** [comment]
AFAIK this also affects iOS and should therefore be moved to the cwa-documentation repo.

**Comment by MikeMcC399** [comment]
@heinp 

After an infection has been confirmed, exposure logging should be deactivated and there should be no warnings received from other people. CWA only sends out warnings about an infection to other people who were met BEFORE the infection was confirmed. After the infection is known about, the person is expected to isolate. Once the person has recovered and is mixing with other people again, then exposure logging can be reactivated. See FAQ [How do I tell the app that I recovered from COVID-19?](https://www.coronawarn.app/en/faq/#recovered).

- This is related to an open FAQ request https://github.com/corona-warn-app/cwa-website/issues/841.

**Comment by Ein-Tim** [comment]
@MikeMcC399

The problem is that even when a positive test is in the app and then the risk status changes from increased to low, the pop up about this is shown ("Ihr Risikostatus hat sich geändert...").

**Comment by MikeMcC399** [comment]
@heinp 

Could you say whether exposure logging was active or not when the low risk warning appeared?

**Comment by heinp** [comment]
> @heinp
> 
> Could you say whether exposure logging was active or not when the low risk warning appeared?

In deed it was.
Still, I don't think most users read the faq, so eg exposure logging should be deactivated automatically after a positive test.

To be clear: it was no new low risk warning but but a notification that 14 days have past since the contact that caused the high risk warning and therefore there should be a low risk.

My concern is, that people read this message  too fast and think that their quarantine is over.
